example_scripts/example_runScripts.py

```diff
@@ -21,20 +22,57 @@
 fitbenchmarking_folder = os.path.abspath(os.path.join(current_path, os.pardir))
 scripts_folder = os.path.join(fitbenchmarking_folder, 'fitbenchmarking')
 sys.path.insert(0, scripts_folder)
+sys.path.insert(1, fitbenchmarking_folder)
```

Comment:
Probably not necessary

Reply:
The sys.path calls are being removed in #240